frontend/src/metabase/internal/lib/components-webpack.js

```diff
@@ -22,4 +24,16 @@ const components = [
   ...getComponents(containersReq),
 ];
 
+// provide some stats on the total vs total documented components
+const documented = getComponents(componentsReq).length;
+
+// get everything and then subtract documented
+const total = getComponents(allComponentsReq).length - documented;
+
+export const stats = {
+  total,
+  documented,
+  ratio: total / documented,
```

Comment:
If we're computing `total` to count the components, I think this should be `documented / total`.